tools/sigmut/sigmut_macros.xml

```diff
@@ -0,0 +1,28 @@
+<macros>
+      
+    <token name="@VERSION@">11.19</token>
+
+    <token name="@pipefail@"><![CDATA[set -o | grep -q pipefail && set -o pipefail;]]></token>
+
+    <xml name="requirements">
+        <requirements>
+		<requirement type="package" version="1.17.5">numpy</requirement>
+		<requirement type="package" version="1.3.1">scipy</requirement>
+		<requirement type="package" version="0.25.3">pandas</requirement>
+		<requirement type="package" version="0.10.1">statsmodels</requirement>
+		<requirement type="package" version="3.1.2">matplotlib</requirement>
+		<requirement type="package" version="1.0">sigmut</requirement>
```

Comment:
you can try to remove all requirements except sigmut. I think they are already included in sigmut.

Reply:
Ok, I will do this and push PR.